R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
+#'
+#' Construct XML data structure for \code{modeltime.xml}.
```

Comment:
FWIW, I like the split design too.  Many of these comments reflect the fact that I was figuring out how the system works as I was going through it.